composer.json

```diff
@@ -23,7 +23,7 @@
         "knplabs/gaufrette": "^0.1.6 || ^0.2",
         "kriswallsmith/buzz": "^0.15",
         "sonata-project/classification-bundle": "^3.0",
-        "sonata-project/core-bundle": "^3.0",
+        "sonata-project/core-bundle": "^3.2",
```

Comment:
Because on older version we get this when running lower-deps:

Error: Call to protected method PHPUnit_Framework_TestCase::getMockForAbstractClass() from context 'Sonata\CoreBundle\Test\EntityManagerMockFactory'